dashboards.d/matrix_epfl_ch.json

```diff
@@ -340,5 +736,5 @@
   "timezone": "",
   "title": "matrix.epfl.ch",
   "uid": "7aM4y6_Sk",
-  "version": 0
-}
+  "version": 4
+}
```

Comment:
You removed the empty line at EOF

Reply:
thanks, but I didn't change the json at all - it's how it is exported by Grafana :woman_shrugging: I prefer not changing them by hand/linter to avoid breaking them being importable by Grafana